src/fuse32.rs

```diff
@@ -0,0 +1,168 @@
+//! Implements Fuse32 filters.
+
+use crate::{fuse_contains_impl, fuse_from_impl, Filter};
+use alloc::{boxed::Box, vec::Vec};
+use core::convert::TryFrom;
+
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+/// Xor filter using 32-bit fingerprints in a [fuse graph]. Requires less space than an [`Xor32`].
+///
+/// A `Fuse32` filter uses <36.404 bits per entry of the set is it constructed from, and has a false
+/// positive rate of <0.001%. As with other probabilistic filters, a higher number of entries decreases
```

Comment:
honestly maybe we can just say
```suggestion
/// A `Fuse32` filter uses <36.404 bits per entry of the set is it constructed from, and has a false
/// positive rate of effectively zero (1/2^32 =~ 1/4 billion). As with other probabilistic filters, a higher number of entries decreases
```